docs/source/deployment/kubernetes/corda-kubernetes-deployment-option.rst

```diff
@@ -0,0 +1,175 @@
+Corda Kubernetes Deployment option
+==================================
+
+Now that we have reviewed the architecture and the prerequisites we can start talking about the actual deployment scripts.
+
+The scripts are located at: `Github: Corda Deployments <https://github.com/corda/corda-deployments>`_
+
+The scripts come with usage instructions of their own, but it is worth explaining the steps in more details.
+
+There are a few distinct steps to deploying a Corda Node that are always required.
+
+* The first step is registration / certificates generation.
+* Then moving on to deployment
+* And finally starting everything up and testing
+
+Registration / certificates generation
```

Comment:
Suggestion : all upper case for this Registration & Certificate Generation

Suggestion : To view the full process point the user to https://solutions.corda.net/deployment/node-registration.html